largest_sum_contiguous_subarray/largestSumContiguousSubarray.c

```diff
@@ -1,7 +1,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-int largestSumContinousSubArray(int arr[], int size) {
+int largestSumContinousSubarray(int arr[], int size) {
```

Comment:
@jsroyal Sir, Revert all changes in `largestSumContiguousSubarray.c` in **Branch -`jsroyal:fibonacci`**